publish-app-do/index.md

```diff
@@ -0,0 +1,458 @@
+
+```bash
+adduser appuser
+```
+
+When prompted, enter and confirm a password for `appuser`. 
+
+After the password is successfully updated, you are prompted to change the user information for appuser. Press ENTER to use the default values. Type `Y` to confirm the information is correct.
+
+To give appropriate permissions, add `appuser` to the `sudo` group:
+
+```bash
+usermod -aG sudo appuser
+```
+
+See [How To Create a New Sudo-enabled User on Ubuntu 18.04](https://www.digitalocean.com/community/tutorials/how-to-create-a-new-sudo-enabled-user-on-ubuntu-18-04-quickstart).
+
+To be able to login as `appuser`, copy your ssh public key to the new home directory for `appuser`:
+
+```bash
+rsync --archive --chown=appuser:appuser ~/.ssh /home/appuser
+```
+
+Now, switch to your new user account:
+
+```bash
+su - appuser
+```
+
+### Set Up Your Server 
+
+To set up your server to install and run the Starport app, you need to install a few tools to help us to publish your blockchain app.
+
+1. Install [Go](https://golang.org/dl/).
+
+    At the time of writing the tutorial, the latest version of Go is 1.16.5.
+
+    ```bash
+    cd ~
+    wget https://golang.org/dl/go1.16.5.linux-amd64.tar.gz
+    sha256sum go1.16.5.linux-amd64.tar.gz
+    # b12c23023b68de22f74c0524f10b753e7b08b1504cb7e417eccebdd3fae49061  go1.16.5.linux-amd64.tar.gz
+    ```
+
+1. Extract the archive and keep it in `/usr/local`:
+
+    ```bash
+    sudo tar -xvf go1.16.5.linux-amd64.tar.gz -C /usr/local
+    ```
+
+    A directory named `go` is now in the `/usr/local` directory.
+    
+1. Next, recursively change this directory’s owner and group to `appuser`:
+
+    ```bash
+    sudo chown -R appuser:appuser /usr/local/go
+    ```
+
+1. Add the go paths to your profile:
+
+    ```bash
+    nano ~/.profile
+    ```
+
+    At the end of the file, add the following two lines:
+
+    ```bash
+    export GOPATH=$HOME/go
+    export PATH=$PATH:$GOPATH/bin:/usr/local/go/bin
+    ```
+
+1. Update your profile:
+
+    ```bash
+    source ~/.profile
+    ```
+
+1. Verify your go installation:
+
+    ```bash
+    go version
+    ```
+
+    The results:
+
+    ```bash
+    go version go1.16.5 linux/amd64
+    ```
+
+1. To use Starport, install the Starport CLI:
+
+    ```bash
+    curl https://get.starport.network/starport! | sudo bash
+    ``` 
+    The latest stable version of Starport is installed. 
+
+## Upload Your App to the Droplet
+
+Several methods are available to upload your app to the Droplet.
+
+One convenient method is to upload the app to GitHub and then download it on your Droplet. Starport creates a GitHub repository that makes it convenient to add your app to your repositories.
+
+To upload the files, you can use `scp` to avoid a few extra steps. 
+
+1. In a terminal window on your local machine, use the following command to upload the local app directory to your DigitalOcean server:
+
+    `scp -r <appname> appuser@<IP address>:/home/appuser/`
```

Comment:
@toschdev I wonder if we should just use `digitalocean` throughout, since we use specific examples later in the tutorial?

Reply:
oh yes, totally 👍